counsel.el

```diff
@@ -2466,7 +2466,14 @@ It applies no filtering to ivy--all-candidates."
     (insert (format "-*- mode:grep; default-directory: %S -*-\n\n\n" default-directory))
     (insert (format "%d candidates:\n" (length ivy--all-candidates)))
     (ivy--occur-insert-lines
-     (mapcar (lambda (cand) (concat "./" prepend cand)) ivy--all-candidates))))
+     (mapcar (lambda (cand)
+               (save-match-data
+                 (if (string-match
+                      ;; Remove column indicators if present
+                      "\\(?1:[^\n:]+?[^\n/:]\\):[\t ]*\\(?3:[1-9][0-9]*\\)[\t ]*:\\(?3:[1-9][0-9]*:\\)" cand)
+                     (replace-match "" nil nil cand 3)
+                   cand)
+               (concat "./" prepend cand))) ivy--all-candidates))))
```

Comment:
Nitpick: Please place `ivy--all-candidates` on a separate line, where it won't be hidden.